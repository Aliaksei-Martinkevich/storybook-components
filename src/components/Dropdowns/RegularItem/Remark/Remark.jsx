import React from 'react';
import classNames from 'classnames';
import PropTypes from 'prop-types';

import './Remark.scss';

const Remark = ({ text, className }) => (
  <span className={classNames('remark', className)}>
    {text}
  </span>);

Remark.defaultProps = {
  className: '',
};

Remark.propTypes = {
  text: PropTypes.string.isRequired,
  className: PropTypes.string,
};

export default Remark;
