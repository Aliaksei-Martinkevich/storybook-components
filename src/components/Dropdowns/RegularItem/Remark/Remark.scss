.remark {
    font-size: 12px;
    line-height: 16px;
    letter-spacing: 0.3;
    color: #BBBBBB;
}