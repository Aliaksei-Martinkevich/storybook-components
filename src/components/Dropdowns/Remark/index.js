import React from 'react';
import PropTypes from 'prop-types';

import './index.css';

const Remark = ({ text }) => (<span className='remark'>{text}</span>);

Remark.propTypes = {
  text: PropTypes.string,
};

export default Remark;
