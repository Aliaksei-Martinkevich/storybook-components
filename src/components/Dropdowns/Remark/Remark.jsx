import React from 'react';
import PropTypes from 'prop-types';

import './Remark.scss';

const Remark = ({ text }) => (<span className="remark">{text}</span>);

Remark.propTypes = {
  text: PropTypes.string,
};

export default Remark;
