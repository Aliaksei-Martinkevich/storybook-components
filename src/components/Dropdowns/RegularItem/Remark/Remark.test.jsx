import React from 'react';
import renderer from 'react-test-renderer';
import Remark from './Remark.jsx';

describe('Remark', () => {
  it('Should render without fails', () => {
    const customClassName = 'class-name-example';
    const text = 'Some text';
    const component = renderer.create(<Remark className={customClassName} text={text} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
