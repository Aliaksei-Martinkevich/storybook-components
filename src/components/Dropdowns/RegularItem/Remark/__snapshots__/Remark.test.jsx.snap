// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Remark Should render without fails 1`] = `
<span
  className="remark class-name-example"
>
  Some text
</span>
`;
